components/romio/src/ome/io/nio/SimpleBackOff.java

```diff
@@ -48,13 +48,19 @@
 
     protected final TileSizes sizes;
 
+    protected final long maxPixels;
+    
+    protected final long defaultValue;
+    
     public SimpleBackOff() {
-        this(new ConfiguredTileSizes());
+        this(new ConfiguredTileSizes(), 1000, 1000000);
     }
 
-    public SimpleBackOff(TileSizes sizes) {
+    public SimpleBackOff(TileSizes sizes, long defaultValue, long maxPixels) {
```

Comment:
keep the method ``public SimpleBackOff(TileSizes sizes)`` so it does not break anything and introduce this one